src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/job/AggregatedVcfJobParametersValidator.java

```diff
@@ -50,6 +51,9 @@ private CompositeJobParametersValidator compositeJobParametersValidator(JobParam
 
         Boolean skipAnnotation = Boolean.valueOf(jobParameters.getString(JobParametersNames.ANNOTATION_SKIP));
         if (!skipAnnotation) {
+            // the next validator is required only in some jobs that use the annotation steps, the steps don't include it
```

Comment:
Please clarify why **this** job needs it. Applies to the other VCF processing job.